src/Version.php

```diff
@@ -17,6 +17,9 @@
 
 class Version implements JsonSerializable
 {
+    /** @var ComparatorInterface */
+    private static $comparator;
```

Comment:
Please put declaration of this property below `protected $build`. I like properties methods to be ordered by their significance and relevance to the class they belong to.